pkg/cluster/mpi/mpi.go

```diff
@@ -42,8 +42,8 @@ const (
 	// Period of rerunning resource finalizers
 	finalizerRetryPeriod = 1 * time.Second
 
-	sidecarImage    = "quay.io/ddl_apetrov/rsync-sidecar:0.0.5" // TODO
-	customizerImage = "quay.io/ddl_apetrov/rsync-sidecar:0.1.7" // TODO
+	defaultInitImage = "quay.io/ddl_apetrov/rsync-sidecar:0.1.7" // TODO
+	defaultSyncImage = "quay.io/ddl_apetrov/rsync-sidecar:0.0.5" // TODO
```

Comment:
Don't forget to change these

Reply:
@JoshBroomberg : I agree that using subdirecttories would be nice, but the current actions apparently do not support it (in https://github.com/marketplace/actions/build-and-push-docker-images#git-context ... ⚠️ Subdirectory for Git context is not yet available for the buildx docker driver).

Reply:
Agree on vendored images.